demos/blockfactory/blocks.js

```diff
@@ -228,6 +228,24 @@ Blockly.Blocks['input_dummy'] = {
   }
 };
 
+Blockly.Blocks['input_end_row'] = {
+  // End-row input.
+  init: function() {
+    this.jsonInit({
+      "message0": "end-row input",
+      "message1": FIELD_MESSAGE,
+      "args1": FIELD_ARGS,
+      "previousStatement": "Input",
+      "nextStatement": "Input",
+      "colour": 210,
+      "tooltip": "For adding fields at the end of a row with no " +
```

Comment:
Nit: Update tooltip.

Reply:
I updated the tooltips for both this and dummy inputs, please take a look.